### PR TITLE
Add access and authentication page for Google doc

### PR DESCRIPTION
**Pull request by dstrain115**
- Access and Authentication page that describes projects
and users and gives some hints for potential issues.

This is needed for the new website where all vendors will
have an access and auth page to clarify access requirements.

### CODE REVIEW

File: docs/_book.yaml [hunk 1]
```diff
@@ -73,6 +73,8 @@ upper_tabs:
 
       ####  GOOGLE HARDWARE ####
       - heading: "Google hardware"
+      - title: "Access and Authentication"
```
Comment (balopat) [inline docs/_book.yaml:76]: Sentence-case is the devsite recommendation: 

```suggestion
      - title: "Access and authentication"
```

File: docs/google/access.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+in partnership with a Google sponsor.
+
+Access and authentication for Google Hardware is done through Google Cloud
+Platform.  In order to use Google Hardware through the Quantum Computing
+Service, you will need a Google account and a Google Cloud Project.
+There are two main categories of these accounts:
+
+*   Personal accounts, such as a gmail.com account and a personally created
+project.
+*   Google enterprise (or gSuite) account.  These accounts are typically
```
Comment (balopat) [inline docs/google/access.md:14]: I think they renamed it: 

```suggestion
*   Google enterprise (or Google Workspace) account.  These accounts are typically
```

File: docs/google/access.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## User access
+
+Each user (i.e. email address) will need to be associated with the project
+and have appropriate IAM permissions on the project in order to access
+the API through this project.
+
+Each user will also have to be added to an approved list in order to access
+the Quantum Computing Service.  Please submit any new users to your Google
+sponsor so that they can be added.
```
Comment (balopat) [inline docs/google/access.md:55]: Can we add a little snippet / pointer to the Getting started guide to test the access? How do I know it works?

**Review by balopat — APPROVED** [review]
This is great, thank you! Added some nits!

File: docs/google/access.md (hunk 3 above)
Comment (dstrain115) [reply docs/google/access.md:55]: Added a next steps section.

File: docs/google/access.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Next Steps
+
+At this point, you should now have access to the Quantum Computing Service.
+You can try out our [Getting Started Guide](../tutorials/google/start.ipyn).
```
Comment (balopat) [inline docs/google/access.md:60]:
```suggestion
You can try out our [Getting Started Guide](../tutorials/google/start.ipynb).
```

**Pull request merged**